<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatricePolonaise</class>
 <widget class="QMainWindow" name="CalculatricePolonaise">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalculatricePolonaise</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLineEdit" name="affichageConstante">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxClavier">
      <property name="text">
       <string>Afficher le clavier</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Pile</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="affichagePile">
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layoutClavier">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Opérations générales</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QPushButton" name="pushButtonCancel">
               <property name="text">
                <string>CANCEL</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonPush">
               <property name="text">
                <string>PUSH</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonExp">
               <property name="text">
                <string>EXPRESSION</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Opérations sur la pile</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QGridLayout" name="gridLayout_5">
               <property name="horizontalSpacing">
                <number>6</number>
               </property>
               <item row="0" column="1">
                <widget class="QPushButton" name="pushButtonSwap">
                 <property name="text">
                  <string>SWAP</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="pushButtonSum">
                 <property name="text">
                  <string>SUM</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QPushButton" name="pushButtonMean">
                 <property name="text">
                  <string>MEAN</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QPushButton" name="pushButtonClear">
                 <property name="text">
                  <string>CLEAR</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="pushButtonDup">
                 <property name="text">
                  <string>DUP</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QPushButton" name="pushButtonDrop">
                 <property name="text">
                  <string>DROP</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Opérations</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="QPushButton" name="pushButtonPow">
                 <property name="text">
                  <string>POW</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="pushButtonMod">
                 <property name="text">
                  <string>MOD</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QPushButton" name="pushButtonSign">
                 <property name="text">
                  <string>SIGN</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="pushButtonSin">
                 <property name="text">
                  <string>SIN</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="pushButtonCos">
                 <property name="text">
                  <string>COS</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QPushButton" name="pushButtonTan">
                 <property name="text">
                  <string>TAN</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QPushButton" name="pushButtonSinh">
                 <property name="text">
                  <string>SINH</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QPushButton" name="pushButtonCosh">
                 <property name="text">
                  <string>COSH</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QPushButton" name="pushButtonTanh">
                 <property name="text">
                  <string>TANH</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QPushButton" name="pushButtonLn">
                 <property name="text">
                  <string>LN</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QPushButton" name="pushButtonLog">
                 <property name="text">
                  <string>LOG</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="2">
                <widget class="QPushButton" name="pushButtonInv">
                 <property name="text">
                  <string>INV</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QPushButton" name="pushButtonSqrt">
                 <property name="text">
                  <string>SQRT</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QPushButton" name="pushButtonSqr">
                 <property name="text">
                  <string>SQR</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="2">
                <widget class="QPushButton" name="pushButtonCube">
                 <property name="text">
                  <string>CUBE</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QPushButton" name="pushButtonFact">
                 <property name="text">
                  <string>!</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1" colspan="2">
                <widget class="QPushButton" name="pushButtonEval">
                 <property name="text">
                  <string>EVAL</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QPushButton" name="pushButton7">
                 <property name="text">
                  <string>7</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="pushButton8">
                 <property name="text">
                  <string>8</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QPushButton" name="pushButton9">
                 <property name="text">
                  <string>9</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QPushButton" name="pushButtonDiv">
                 <property name="text">
                  <string>/</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="pushButton4">
                 <property name="text">
                  <string>4</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QPushButton" name="pushButton6">
                 <property name="text">
                  <string>6</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="pushButton5">
                 <property name="text">
                  <string>5</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QPushButton" name="pushButton3">
                 <property name="text">
                  <string>3</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QPushButton" name="pushButton2">
                 <property name="text">
                  <string>2</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QPushButton" name="pushButton1">
                 <property name="text">
                  <string>1</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QPushButton" name="pushButton0">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="2">
                <widget class="QPushButton" name="pushButtonVirgule">
                 <property name="text">
                  <string>,</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <widget class="QPushButton" name="pushButtonMult">
                 <property name="text">
                  <string>*</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="3">
                <widget class="QPushButton" name="pushButtonMoins">
                 <property name="text">
                  <string>-</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="3">
                <widget class="QPushButton" name="pushButtonPlus">
                 <property name="text">
                  <string>+</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QPushButton" name="pushButtonDollar">
                 <property name="text">
                  <string>$</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Mode constante</string>
            </property>
            <widget class="QWidget" name="verticalLayoutWidget">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>101</width>
               <height>68</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QRadioButton" name="radioButtonEntier">
                <property name="text">
                 <string>Entier</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButtonRationnel">
                <property name="text">
                 <string>Rationnel</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButtonReel">
                <property name="text">
                 <string>Réel</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Complexes</string>
            </property>
            <widget class="QWidget" name="verticalLayoutWidget_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>181</width>
               <height>71</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QRadioButton" name="radioButtonAvecComplexes">
                <property name="text">
                 <string>Avec</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButtonSansComplexes">
                <property name="text">
                 <string>Sans</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>612</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionA_propos"/>
    <addaction name="actionA_propos_de_Qt"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionAnnuler"/>
    <addaction name="actionR_tablir"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menu"/>
  </widget>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionA_propos_de_Qt">
   <property name="text">
    <string>A propos de Qt</string>
   </property>
  </action>
  <action name="actionAnnuler">
   <property name="text">
    <string>Annuler</string>
   </property>
  </action>
  <action name="actionR_tablir">
   <property name="text">
    <string>Rétablir</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>CalculatricePolonaise</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
